vagga.yaml

```diff
@@ -0,0 +1,32 @@
+minimum-vagga: v0.7.0
+containers:
+   pip-freeze:
+     setup:
+       - !Alpine v3.4
+       - !Py3Install
+         - pip
+         - 'Django >=1.9,<1.10'
+         - 'gunicorn >=19.6,<20.0'
+         - 'whitenoise >=3.2,<3.3'
+         - 'django-environ >=0.4,<0.5'
+
+       - !BuildDeps
+         - postgresql-dev
```

Comment:
Sometime it matters where this put. I recommend you to put it above `!Py3Install`